addrmgr/addrmanager.go

```diff
@@ -1206,7 +1213,7 @@ func (a *AddrManager) GetBestLocalAddress(remoteAddr *wire.NetAddressV2) *wire.N
 // Use Start to begin processing asynchronous address updates.
 func New(dataDir string, lookupFunc func(string) ([]net.IP, error)) *AddrManager {
 	am := AddrManager{
-		peersFile:      filepath.Join(dataDir, "peers.json"),
+		store:          NewStore(filepath.Join(dataDir, "peers.json")),
```

Comment:
Should the store also be accepted as an optional argument so it can be properly initialized in the WASM/browser context?

Reply:
Ah scratch that, I see the build tag usage below!